experiment/test_reporter.py

```diff
@@ -16,24 +16,37 @@
 import os
 from unittest import mock
 
+import yaml
+
 from experiment import reporter
 from test_libs import utils as test_utils
 
 # pylint: disable=unused-argument,invalid-name
 
 
-def test_output_report_bucket(fs, experiment):
-    """Test that output_report writes the report and rsyncs it to the web
-    bucket."""
-    web_bucket = 'gs://web-bucket/experiment'
+def test_output_report_filestore(fs, experiment):
+    """Test that output_report writes the report and rsyncs it to the report
+    filestore."""
+    # Get the config.
+    config_filepath = os.path.join(os.path.dirname(__file__), 'test_data',
+                                   'experiment-config.yaml')
+    fs.add_real_file(config_filepath)
```

Comment:
Do you need this, since we are reading the experiment config yaml content below.

Reply:
I do need this because we are using the fakefs and can't read it otherwise.